### DISCUSSION TITLE
dunst -print does nothing

### DISCUSSION THREAD

**Discussion by GoodDays13** [dunst-project/dunst#1318] [Help] [answered]
I'm on Arch, and when I use `-print` or `--print` it doesn't print anything on a notification, except `CRITICAL: Source ID ## was not found when attempting to remove it`

**Comment by bynect** [comment, marked as answer]
Hello, this was recently resolved by #1299

**Reply by yonbi-ops** [reply]
me too I use the ChromeOS dev env (I know it runs DEBIN) so when I do dunset -print it outputs nothing can you fix the apt verson
